prow/plugins/golint/golint.go

```diff
@@ -232,6 +251,9 @@ func handle(minimumConfidence float64, ghc githubClient, gc *git.Client, log *lo
 
 	// Make the list of comments.
 	var comments []github.DraftReviewComment
+	if lintErrorComment != (github.DraftReviewComment{}) {
```

Comment:
Please make `problemsInFiles()` return `*DraftReviewComment` and check against nil to determine whether there's a comment